tests/snuba/api/endpoints/test_project_tags.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+
+from sentry.testutils import (
+    APITestCase,
+    SnubaTestCase,
+)
+
+
+class ProjectTagsTest(APITestCase, SnubaTestCase):
```

Comment:
I just moved this into snuba tests and slightly modified. We really need to get most of our tests running with the SnubaTagStorage backend, since we're not actually testing production behaviour properly at this point.